libs/libcommon/src/libcommon/viewer_utils/asset.py

```diff
@@ -42,16 +55,12 @@ def update_last_modified_date_of_rows_in_assets_dir(
     length: int,
     assets_directory: StrPath,
 ) -> None:
+    namespace, repo = dataset.split("/")
+    root_dir = namespace if repo else dataset
+    update_directory_modification_date(Path(assets_directory).resolve() / root_dir)
```

Comment:
this hsould do the job
```suggestion
    update_directory_modification_date(Path(assets_directory).resolve() / dataset.split("/")[0)
```